### PR TITLE
Meta refresh redirect support 

### PR DESCRIPTION
**Pull request by logandhead**
I made an optional argument in the confugration that supports redirects that are stated in meta_refresh tags in accordance to these specs https://www.w3.org/TR/WCAG20-TECHS/H76.html and this issue https://github.com/codelucas/newspaper/issues/260

### CODE REVIEW

**Comment by codelucas** [comment]
Thanks for the PR! This mostly looks good to me but I'll let @yprez give the final call on merging this

**Comment by codelucas** [comment]
If he doesn't respond in a few days I'll merge

**Comment by adamn** [comment]
I upped the version to 0.1.8 so we could install on our systems more easily.  I'm happy to switch that around or revert.

File: tests/unit_tests.py [hunk 1]
```diff
@@ -179,6 +179,17 @@ def test_download_html(self):
...
     @print_test
+    def test_meta_refresh_redirect(self):
+        config = Configuration()
+        config.follow_meta_refresh = True
+        article = Article(
+            '', config=config)
+        html = mock_resource_with('google_meta_refresh', 'html')
+        article.download(html=html)
+        article.parse()
+        self.assertEqual(article.title, 'Example Domain')
```
Comment (yprez) [inline tests/unit_tests.py]: Does this actually pull from example.com? I'm not sure it's such a good idea to do that during tests, but can't think of other alternatives that will fit into the current mocking pattern...

Comment (yprez) [inline tests/unit_tests.py]: I guess we can stick a TODO comment here for now and merge this.

Comment (adamn) [inline tests/unit_tests.py]: TODO is now there.

**Pull request merged**